examples/custom-field/3-pair-field-json/index.ts

```diff
@@ -44,6 +44,7 @@ export function pair<ListTypeInfo extends BaseListTypeInfo>(
   config: PairFieldConfig<ListTypeInfo> = {}
 ): FieldTypeFunc<ListTypeInfo> {
   function resolveInput(value: PairInput | null | undefined) {
+    if (!value) return undefined;
```

Comment:
@gathanase thanks for making the changes, but I think this might actually want to be specifically `if (value === undefined)`?
`null` is actually a value we might want to represent, whereas `undefined` means "unchanged".

Reply:
If the user want to put null in database then we should return null if value is null. In any case, I think you are right that we should only consider undefined.